bwakit/Dockerfile

```diff
@@ -2,11 +2,17 @@ FROM ubuntu
 
 MAINTAINER Frank Austin Nothaft, fnothaft@berkeley.edu
 
-RUN apt-get update && apt-get install -y curl bzip2 perl
+RUN apt-get update && apt-get install -y curl bzip2 perl make g++
 
 # pull down bwa-kit
-RUN curl -L http://sourceforge.net/projects/bio-bwa/files/bwakit/bwakit-0.7.12_x64-linux.tar.bz2 \
-    | tar -xjC /opt/ 
+RUN curl -L http://sourceforge.net/projects/bio-bwa/files/bwakit/bwakit-0.7.15_x64-linux.tar.bz2 \
+    | tar -xjC /opt/
+
+# get latest version of samblaster and build
+RUN curl -L https://github.com/GregoryFaust/samblaster/releases/download/v.0.1.24/samblaster-v.0.1.24.tar.gz \
```

Comment:
samblaster (specifically this version which is later than the latest bwakit version) is needed by TOPMed